api/stats.proto

```diff
@@ -0,0 +1,150 @@
+// [#protodoc-title: Stats sinks]
```

Comment:
Nit: not just sinks now.

Reply:
Right. 0fbe7b314a135a23bb05a19cb2d3a66d6b5a1be7